src/test/rustdoc/const-generics/const-impl.rs

```diff
@@ -1,3 +1,5 @@
+// ignore-tidy-linelength
```

Comment:
This isn't needed, it's implied in tests.

Reply:
This was because of the `@` typo.